plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -54,5 +54,37 @@
       markerType="com.google.cloud.tools.eclipse.appengine.validation.blacklistMarker">
     </markerResolutionGenerator>
   </extension>
+  
+    <extension 
+      name="Web Xml Validator" 
+      point="org.eclipse.wst.validation.validatorV2" 
+      id="webXmlValidator">
+    <validator
```

Comment:
Should we define a `markerId` attribute on the `<validator>`?  I suspect this may do something with showing the markers in a separate category?